projects/plugins/jetpack/modules/related-posts/jetpack-related-posts.php

```diff
@@ -408,12 +409,26 @@ public function render_block( $attributes ) {
 			$rows_markup .= $this->render_block_row( $lower_row_posts, $block_attributes );
 		}
 
-		return sprintf(
+		$display_markup = sprintf(
 			'<nav class="jp-relatedposts-i2" data-layout="%1$s">%2$s%3$s</nav>',
 			esc_attr( $block_attributes['layout'] ),
 			$block_attributes['headline'],
 			$rows_markup
 		);
+
+		/**
+		 * Filter the output HTML of Related Posts.
+		 *
+		 * @module related-posts
+		 *
+		 * @since $$next-version$$
+		 *
+		 * @param string $display_markup HTML output of Related Posts.
+		 * @param string get_the_ID() Post ID of the post for which we are retrieving Related Posts.
```

Comment:
`get_the_ID()` [returns an int or false](https://developer.wordpress.org/reference/functions/get_the_id/), so I would update the documentation accordingly:

```suggestion
		 * @param int|false get_the_ID() Post ID of the post for which we are retrieving Related Posts.
```